recipe/meta.yaml

```diff
@@ -1,82 +1,70 @@
 {% set version = "1.0" %}
-{% set build = 202 %}
-{% set variant = "openblas" %}
-{% set sha256 = "71d3c9eb8a183150f965a465824d01fe82826c22505f7aa314f700ace03fa77f" %}
 
 package:
   name: caffe
   version: {{ version }}
 
 source:
-  fn: caffe_{{ version }}.tar.gz
   url: https://github.com/BVLC/caffe/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 71d3c9eb8a183150f965a465824d01fe82826c22505f7aa314f700ace03fa77f
+  patches:
+    - backport_6638-support-opencv4.patch
 
 build:
-  number: {{ build }}
-  skip: true  # [not (linux and py27)]
-  features:
-    - blas_{{ variant }}
+  number: 203
+  detect_binary_files_with_prefix: False
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - boost
     - cmake
```

Comment:
I think cmake should be in ``build`` right?

Reply:
I'm not sure that makes any difference but it is worth trying.